apollo-gradle-plugin/src/main/java/com/apollographql/apollo/gradle/ApolloExtension.java

```diff
@@ -12,6 +12,7 @@
   private Map<String, String> customTypeMapping = new LinkedHashMap<>();
   private String nullableValueType = NullableValueType.ANNOTATED.getValue();
   private boolean generateAccessors = true;
+  private boolean useSemanticNaming = false;
```

Comment:
This makes `useSemanticNaming` default to false. Wouldn't mind  making this true, if we want to default to same behavior as iOS.